aten/src/ATen/native/native_functions.yaml

```diff
@@ -3583,10 +3587,12 @@
   matches_jit_signature: True
   python_module: nn
 
-- func: adaptive_max_pool3d_out(Tensor output, Tensor indices, Tensor self, IntList[3] output_size) -> (Tensor output, Tensor indices)
+# Return: (Tensor output, Tensor indices)
+- func: adaptive_max_pool3d_out(Tensor output, Tensor indices, Tensor self, IntList[3] output_size) -> (Tensor, Tensor)
```

Comment:
why did some of these returns lose their names?

Reply:
We hadn't finalized whether or not these were the names we wanted for these returns, so they were commented out for now, to be readded when we agreed on them.